aiida/tools/importexport/dbexport/zip.py

```diff
@@ -99,9 +103,12 @@ def close(self):
     def pwd(self):
         return self._pwd
 
-    def open(self, fname, mode='r'):
+    def open(self, fname, mode='r', encoding='utf8'):
+        if 'b' in mode:
+            encoding = None
+
         if mode == 'w':
```

Comment:
This should rather be `if 'w' in mode` because now the `wb` mode will go to the else